lib/openshift/role_binding.rb

```diff
@@ -21,9 +21,7 @@ class RoleBinding < ProjectResource
     # - kind: ServiceAccount
     #   name: deployer
     #   namespace: user1-test-prj
-    # - kind: SystemUser
     #   name: system:admin
-    # - kind: SystemGroup
```

Comment:
I would suggest to change `SystemGroup/User` to `Group/User` instead of removing the lines. Otherwise the comment becomes confusing.